docs/StandaloneConfiguration.md

```diff
@@ -27,6 +28,11 @@
 ```
 "ignoreGlobList": ["**.dat", "**.js"]  <-- Repository level's list of file formats to ignore.
```

Comment:
We should update the example here to demonstrate how both can use in conjunction with each other.
Let's change the examples to folder ignore instead.